.eslintrc.json

```diff
@@ -0,0 +1,52 @@
+{
+  "env": {
+    "node": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:react-hooks/recommended",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+    "plugin:jsx-a11y/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "eslint-config-prettier",
+    "prettier"
+  ],
+  "plugins": ["react", "prettier"],
+  "settings": {
+    "import/resolver": {
+      "node": {
+        "paths": ["${__dirname}"],
+        "extensions": [".ts", ".tsx"]
+      },
+      "typescript": {}
```

Comment:
Do we need it?

Reply:
I need "typescript" option to make ESLint understand and resolve my import like "import "@styles/index.scss";" or "import { Config } from "tailwindcss/types/config";".
With regard to "paths" and "extentions" options, I have researched but I cannot understand it and whether I implement this or not, my code still work. Can you drop me a hint?